scipy/optimize/_highs/cython/src/_highs_wrapper.pyx

```diff
@@ -705,7 +722,6 @@ def _highs_wrapper(
         res = {
             'status': <int> model_status,
             'message': highs.modelStatusToString(model_status).decode(),
-            'unscaled_status': <int> unscaled_model_status,
```

Comment:
This is unrelated? Can you remind me what this is for and why it's no longer needed?

Reply:
This is related to the note above about deprecation of `getModelStatus(const bool scaled_model)`.  The scaled `model_status` used to be compared to the normal `unscaled_model_status` to know if there was any discrepancies between the scaled problem vs original problem, but this is now absent in the main HiGHS repo and all instances of `scaled_model_status` have been removed.  The function was moved to a deprecated functions file and `unscaled_model_status` will always be equal to `model_status`, so it made sense to me to remove it from here as it adds no information